query/src/sql/exec/mod.rs

```diff
@@ -267,7 +283,7 @@ impl PipelineBuilder {
 
         let mut group_expressions = Vec::with_capacity(aggregate.group_expr.len());
         for scalar_expr in aggregate.group_expr.iter() {
-            let scalar = scalar_expr.as_any().downcast_ref::<Scalar>().unwrap();
```

Comment:
maybe we need to check aggregate functions for group by exprs?

Reply:
good catch!